data/src/main/scala/ackcord/data/message.scala

```diff
@@ -769,3 +778,25 @@ case class OutgoingEmbedAuthor(name: String, url: Option[String] = None, iconUrl
 case class OutgoingEmbedFooter(text: String, iconUrl: Option[String] = None) {
   require(text.length <= 2048, "The footer text of an embed can't have more that 2048 characters")
 }
+
+/**
+  * The structure of a sticker sent in a message.
+  * @param id id of the sticker
+  * @param packId id of the pack the sticker is from
+  * @param name name of the sticker
+  * @param description description of the sticker
+  * @param tags a comma-separated list of tags for the sticker
+  * @param asset sticker asset hash
+  * @param previewAsset sticker preview asset hash
+  * @param formatType	type of sticker format
+  */
+case class Sticker(
+    id: String,
+    packId: String,
+    name: String,
+    description: String,
+    tags: Option[String],
+    asset: String,
+    previewAsset: Option[String],
+    formatType: Int
```

Comment:
Can you make this a proper enumeratum object like most other enums are?

Reply:
Implemented.